lib/puppet/defaults.rb

```diff
@@ -1128,7 +1128,7 @@ def self.initialize_default_settings!(settings)
       :type      => :string,
       :desc      => "Where to send log messages. Choose between 'syslog' (the POSIX syslog
       service), 'eventlog' (the Windows Event Log), 'console', or the path to a log
-      file."
+      file. Multiple destinations can be set using a comma separated list(eg: /path/file1,console,/path/file2)"
```

Comment:
```suggestion
      file. Multiple destinations can be set using a comma separated list (eg: `/path/file1,console,/path/file2`)"
```
This gets rendered as markdown, so might be cool to wrap it in backticks. Also, whitespace before the parenthesis.